persistence/src/test/scala/hmda/persistence/processing/HmdaFileParserSpec.scala

```diff
@@ -26,17 +26,42 @@ class HmdaFileParserSpec extends ActorSpec {
   val badLines = fiCSVParseError.split("\n")
 
   "HMDA File Parser" must {
+    "persist parsed TSs" in {
+      parseTs(lines)
+      probe.send(hmdaFileParser, GetState)
+      probe.expectMsg(HmdaFileParseState(1, Nil))
+    }
+
+    "persist parsed TSs and TS parsing errors" in {
+      parseTs(badLines)
+      probe.send(hmdaFileParser, GetState)
+      probe.expectMsg(HmdaFileParseState(1, Seq(List("Timestamp is not a Long"))))
+    }
+
     "persist parsed LARs" in {
       parseLars(lines)
       probe.send(hmdaFileParser, GetState)
-      probe.expectMsg(HmdaFileParseState(3, Nil))
+      probe.expectMsg(HmdaFileParseState(4, Seq(List("Timestamp is not a Long"))))
```

Comment:
It does produce a valid file, which is why the size goes up by 3.  The `HmdaFileParseState` persists across tests, so the results are cumulative.  The `Seq(List("Timestamp is not a Long"))` is the error from the previous test.

This behavior could probably be changed if I restarted the actor for every test.